src/modules/users/user.module.ts

```
@@ -9,9 +9,10 @@ import { UserUseCase } from './application/use-cases';
 import { SetupModule } from '../setup/setup.module';
 import { AuditModule } from '../audit/audit.module';
 
+@Global()
 @Module({
   controllers: [UserController],
-  exports: [...UserUseCase, UserDomainService, 'UserRepositoryInterface'],
+  exports: [...UserUseCase, UserDomainService, 'UserRepositoryInterface', UserTypeormRepository],
```

ESLint

[error] Replace `...UserUseCase,·UserDomainService,·'UserRepositoryInterface',·UserTypeormRepository` with `⏎····...UserUseCase,⏎····UserDomainService,⏎····'UserRepositoryInterface',⏎····UserTypeormRepository,⏎··`

(prettier/prettier)